build/Common.props

```diff
@@ -23,10 +23,10 @@
       Refer to https://docs.microsoft.com/nuget/concepts/package-versioning for semver syntax.
     -->
     <MinVerPkgVer>[2.3.0,3.0)</MinVerPkgVer>
-    <GoogleProtobufPkgVer>[3.15.5,4.0)</GoogleProtobufPkgVer>
-    <GrpcPkgVer>[2.23.0,3.0)</GrpcPkgVer>
-    <GrpcNetClientPkgVer>[2.32.0,3.0)</GrpcNetClientPkgVer>
-    <GrpcToolsPkgVer>[2.25.0,3.0)</GrpcToolsPkgVer>
+    <GoogleProtobufPkgVer>[3.19.4,4.0)</GoogleProtobufPkgVer>
+    <GrpcPkgVer>[2.44.0,3.0)</GrpcPkgVer>
+    <GrpcNetClientPkgVer>[2.43.0,3.0)</GrpcNetClientPkgVer>
+    <GrpcToolsPkgVer>[2.44.0,3.0)</GrpcToolsPkgVer>
```

Comment:
To fix the M1 build problem, did all of these packages have to be updated?

Reply:
Only updating Grpc.Tools wasn't enough. At that point I went ahead and updated all the Grpc/Protobuf packages thinking it would be better to keep similar packages close together and that they'd all needed updated eventually.

Reply:
I can revert and be more targeted if you'd like.

Reply:
I think it should be ok.